code/mapping/ext_modules/mapping_common/map.py

```
@@ -9,9 +9,10 @@
 from genpy.rostime import Time
 from std_msgs.msg import Header
 from mapping_common import entity
-
+from mapping_common.transform import Transform2D, Point2
 from mapping_common.entity import Entity, FlagFilter, ShapelyEntity
-
+from mapping_common.transform import Vector2, Transform2D
```

Ruff (0.8.2)

Redefinition of unused `Transform2D` from line 12

Remove definition: `Transform2D`

(F811)